doc/manual/syntax.md

```diff
@@ -768,9 +770,9 @@ Examples:
 > {foo.bar = 1, baz = 2} : {foo: {bar : Number}, baz : Number}
 { baz = 2, foo = { bar = 1 } }
 
-># the right-hand side is missing a type annotation for baz, so it isn't a
- # record type
- {foo = 1, bar = "foo" } : {foo : Number, bar : String, baz}
+The right-hand side is missing a type annotation for baz, so it isn't a
```

Comment:
It's a bit strange to mix plain text english inside a seemingly REPL session

Reply:
So, you'd prefer the comments? Or should we try to reformulate the entire example?

Reply:
In fact, I don't know, maybe it's fine. Or maybe as you did (comment before the example), but still prefixing them with `#`? I know it doesn't correspond to anything anymore (it's not really an REPL prompt nor a valid Nickel file nor plain English), but it might at least make the distinction between code, comment and result of the evaluation of the code clearer?

Reply:
I was a bit worried about breaking up the flow of the examples by interspersing prose outside of the code block, but maybe that's not such a bad thing anyway. In fact, I like the rendering on the website with smaller code blocks more.

Reply:
In any case, I just wanted to share my feeling, but I have no strong opinion either and I trust you to keep what you think is best, including the original presentation.